Source/BaseClasses/GDataServiceBase.h

```diff
@@ -21,8 +21,8 @@
   #define GDataUploadFetcherClass GTMSessionUploadFetcher
   #define GDataUploadFetcherClassStr @"GTMSessionUploadFetcher"
 
-  #define kGDataFetcherStatusUnauthorized kGTMSessionFetcherStatusUnauthorized
-  #define kGDataFetcherStatusForbidden kGTMSessionFetcherStatusForbidden
+  #define kGDataFetcherStatusUnauthorized GTMSessionFetcherStatusUnauthorized
+  #define kGDataFetcherStatusForbidden GTMSessionFetcherStatusForbidden
```

Comment:
sorry~
i still not understand what you mean "sync back with the main repo"
i merge master change to my fork, then this pull request increase a commit
Can you give me some tip about sync with main repo?